tests/python/unit/test_ontology_tagger.py

```diff
@@ -87,7 +87,7 @@ def test_matching_in_lowercase():
         language="en",
         normalize_case=True,
     )
-    df = tagger.tag_and_format(
+    df, _ = tagger.tag_and_format(
```

Comment:
I am not a big fan of changing signatures of methods like that. I would prefer to keep the signature of tag_and_format, and introduce a new method to "compute_column_descriptions".